assets/css/custom.css

```diff
@@ -424,7 +424,7 @@ body:last-child .btn-u, x:-moz-any-link {
 .browse-courses {
     display: inline-block;
     border-radius: 6px;
-    margin: 20px 0px 0px 10px;
+    margin: 20px 10px 12px;
```

Comment:
Good concern! I lack the context so I'll make some guess:
* for the sake of consistency (and if all "contact-us" containers have similar purpose) the same styling for the text and link buton could be used on the blog page; moreover, there are indeed two strong sections below blog pagination section, which could be more visually separated: comment/question form and link to courses
* if it really has to be visually different, then it deserves of course a different container style (but keep in mind the inner context will drop in two lines at least on mobile view)